embuilder.py

```diff
@@ -59,33 +59,14 @@
     'libunwind-except'
 ]
 
-USER_TASKS = [
-    'boost_headers',
-    'bullet',
-    'bzip2',
-    'cocos2d',
-    'freetype',
-    'giflib',
-    'harfbuzz',
-    'icu',
-    'libjpeg',
-    'libpng',
-    'ogg',
-    'regal',
+PORTS = sorted(system_libs.ports.ports_by_name.keys()) + [
     'regal-mt',
```

Comment:
maybe add a TODO for the ports to report if they are multitheading-enabled, and then we'd add `-mt` automatically here? (and the other options that cause us to manually list things here)

Reply:
I agree, ports should really declare all the variants they support, not just `-mt`, in some inspect-able way.  

I will add a TODO, and also maybe describe that existing variants as data here rather than code.

Reply:
Much better.  PTAL.